### PR TITLE
update link anatomy, modal cta, tag example

### PR DESCRIPTION
**Pull request by arnoldsandoval-okta**
Updates:

* Link anatomy
* Modal: CTA update, adds `Description` components where necessary
* Tag example

### CODE REVIEW

File: packages/docs/components/modal.md [hunk 1]
```diff
@@ -24,50 +24,6 @@ links:
...
-
-This is where the content of the modal dialog is held. Keep it simple; the need for Tabs or navigation is a warning sign.
-
-### Action bar
-
-Contains the primary action of the modal. All modals must have a primary CTA, and support secondary CTAs as well. Action labels should clarify the task users are being asked to complete.
-
-</Description>
-
-## Behavior
```
Comment (edburyenegren-okta) [inline packages/docs/components/modal.md:51]: I thought we were keeping behavior, just cutting Anatomy notes.

File: packages/docs/components/tag.md [hunk 2]
```diff
@@ -76,35 +76,31 @@ Use Tags as descriptors paired with a particular object. These descriptors may b
           <td>
             Soyuz
             <ul class="ods-tag--list">
-                <li class="ods-tag">Roscosmos</li>
```
Comment (edburyenegren-okta) [inline packages/docs/components/tag.md:79]: Swap to acronyms to account for padding space?

File: packages/docs/components/modal.md (hunk 1 above)
Comment (arnoldsandoval-okta) [reply packages/docs/components/modal.md:51]: Youre right. Will fix.

File: packages/docs/components/tag.md (hunk 2 above)
Comment (arnoldsandoval-okta) [reply packages/docs/components/tag.md:79]: Yeah, we only have so much space in that second column so I made some minor tweaks based on direction from @joejezowski-okta. Starting next week we should resume zoom working sessions to flesh this stuff out so it isn't a surprise like it was this time!

Comment (edburyenegren-okta) [reply packages/docs/components/tag.md:79]: No worries, appreciate the fixes here!

**Review by edburyenegren-okta — APPROVED** [review]
lookin' good!

**Pull request merged**